content/en/docs/v3.3/learning/auth_design.md

```diff
@@ -72,6 +72,8 @@ The auth info in `etcdserverpb.RequestHeader` is checked in the apply phase of t
 
 There are two kinds of token types: simple and JWT. The simple token isn't designed for production use cases. Its tokens aren't cryptographically signed and servers must statefully track token-user correspondence; it is meant for development testing.  JWT tokens should be used for production deployments since it is cryptographically signed and verified. From the implementation perspective, JWT is stateless. Its token can include metadata including username and revision, so servers don't need to remember correspondence between tokens and the metadata.
 
+**Note :** There is a known issue [#18437](https://github.com/etcd-io/etcd/issues/18437) on simple token. Since etcdsever resolves token in the API layer and isn't guarded by linearizable check, and also simple token is stateful. So an etcd member may haven't finished processing previously authentication request before it receives next request, accordingly it will return an "invalid auth token" error back to client.

```

Comment:
Version `3.3.x` is no longer a supported release so the docs for this version are frozen. Please revert your change to this file.

Reply:
reverted the changes